op-node/rollup/derive/batch_queue.go

```diff
@@ -232,6 +232,10 @@ batchLoop:
 	// Fill with empty L2 blocks of the same epoch until we meet the time of the next L1 origin,
 	// to preserve that L2 time >= L1 time
 	if nextTimestamp < nextEpoch.Time {
+		if forceNextEpoch {
```

Comment:
isn't forceNextEpoch always true here? We already checked if it's false, and would have returned. Same for other checks below.